llama_index/vector_stores/types.py

```diff
@@ -143,8 +143,10 @@ def add(
         """Add embedding results to vector store."""
         ...
 
-    def delete(self, doc_id: str, **delete_kwargs: Any) -> None:
-        """Delete doc."""
+    def delete(self, ref_doc_id: str, **delete_kwargs: Any) -> None:
+        """
+        Delete nodes using with ref_doc_id and optionally return ids of deleted nodes
```

Comment:
Checked the impls, don't think we actually return the ids of deleted notes?

Reply:
Right, that was from an original approach lol